### DISCUSSION TITLE
How many repositories are using my GitHub Action?

### DISCUSSION THREAD

**Discussion by ukupat** [community/community#84124] [Actions]
### Select Topic Area

Product Feedback

### Body

It would be very motivating to know how many public + private repositories are using my GitHub action (e.g. [Trello and GitHub integration action](https://github.com/rematocorp/trello-integration-action)). Right now I can only see public repositories from [Dependents view](https://github.com/rematocorp/trello-integration-action/network/dependents).

**Comment by wesleybertipaglia** [comment]
Hi @ukupat, I understand. In fact, it would be great to be able to see the number of repositories that depend on yours or on which your project depends. However, due to security and privacy considerations, GitHub currently does not display private repositories in the dependency graph. This ensures that sensitive information from private repositories is not exposed.

**Reply by ukupat** [reply]
Thank you for the quick response, @wesleybertipaglia! Yes, I totally understand that GitHub can't show private repositories in the Dependents view. What I actually meant was that it would be useful to have a similar graph showing the count of repositories that used the action on a specific day:
<img width="916" alt="Screenshot 2024-01-05 at 11 38 55" src="https://github.com/community/community/assets/4997766/73460d38-758f-4d86-b0f8-2ba6a1b4799c">

This data would be highly motivating for us to continue investing into the open source community.

**Reply by wesleybertipaglia** [reply]
@ukupat amazing, this is a big improvement, in this case you can submit a request through the [GitHub Support Request](https://support.github.com/request). Additionally, for any further questions or assistance, you can reach out to one of the staff members [here](https://github.com/orgs/github/people).

**Reply by ukupat** [reply]
Thanks :) Actually GitHub Support sent me here to write down this request 😅

**Comment by Akash1134** [comment]
Hi @wesleybertipaglia, Thank you for contributing in the community. We really appreciate your inputs and expertise 🙂. 

We wanted to clarify that Github Community is the perfect place for our users to share feedback and raise feature requests. We recommend that users do not reach out to Staff members directly or tag them on a public forum. The community is regularly moderated by Staff members and all your feature requests are valuable for our engineers.

**Reply by wesleybertipaglia** [reply]
@Akash1134 I'm sorry, I won't make this mistake again
